src/v3/pages/reports/meeting_attendance/components/report_toolbar/index.styles.tsx

```diff
@@ -0,0 +1,14 @@
+import { Box } from '@mui/material';
+import { styled } from '@mui/system';
+import { Select } from '@components/index';
+
+export const StyledSelect = styled(Select)({
+  flex: 1,
+  height: '48px',
+});
+
+export const StyledBox = styled(Box)(({ laptopView }: { laptopView: boolean }) => ({
```

Comment:
@Logan08691: please fix this code as well since the `laptoView` prop will produce error in the dev console. Thank you.